StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentMethodFormViewController.swift

```diff
@@ -360,11 +366,17 @@ extension PaymentMethodFormViewController {
                 self.delegate?.updateErrorLabel(for: genericError)
             }
         }
+        let additionalParameters: [String: Any] = [
+            "product": "instant_debits",
+            "attach_required": true,
+            "hosted_surface": "payment_element",
+        ]
```

Comment:
moved from here:

https://github.com/stripe/stripe-ios/pull/3653/files#diff-d34783ba39a9570594c629d85f6d940d3abc6532d8984d12611f9365031305d9L109-L114

Reply:
this is much better implementation because the previous code was actually doing the wrong thing (but there luckily is no impact)